toolkit/tools/imagegen/installutils/installutils.go

```diff
@@ -1062,6 +1062,17 @@ func InstallGrubCfg(installRoot, rootDevice, bootUUID, bootPrefix string, encryp
 	return
 }
 
+func CallGrubMkconfig(installChroot *safechroot.Chroot) (err error) {
+	squashErrors := false
+
+	ReportActionf("Running grub2-mkconfig...")
```

Comment:
As far as I can tell, this isn't doing anything when calling make image...

@dmcilvaney can we take it out?

Reply:
I think this is for ISO installer output.